src/libcmd/repl.cc

```diff
@@ -397,7 +397,7 @@ StringSet NixRepl::completePrefix(const std::string & prefix)
             Expr * e = parseString(expr);
             Value v;
             e->eval(*state, *env, v);
-            state->forceAttrs(v, noPos);
+            state->forceAttrs(v, noPos, "nevermind, it is ignored anyway");
```

Comment:
```suggestion
            state->forceAttrs(v, noPos, ""); // it is ignored anyway
```

Reply:
I prefer my version, as in the event where this strange string gets printed, and not ignored as expected, users have a chance to find the location where it comes from.

Reply:
Then let's just fix the message today. This seems accurate and unique enough.

```suggestion
            state->forceAttrs(v, noPos, "while evaluating an attrset for the purpose of completion");
```

Reply:
```suggestion
            state->forceAttrs(v, noPos, "while evaluating an attrset for the purpose of completion (this error should not be displayed; file an issue?)");
```